ocs_ci/helpers/dr_helpers_ui.py

```
@@ -200,6 +204,159 @@ def verify_drpolicy_ui(acm_obj, scheduling_interval):
+            acm_loc["critical-alert"][0]
+        ).get_attribute("aria-expanded")
+
+        if critical_alert == "false":
+            critical_alert_elem = wait_for_element_to_be_clickable(
+                acm_loc["critical-alert"]
+            )
+            acm_obj.driver.execute_script(
+                "arguments[0].click();", critical_alert_elem
+            )
+            acm_obj.take_screenshot()
+    except Exception as e:
```

Ruff (0.15.14)

[warning] Do not catch blind exception: `Exception`

(BLE001)